src/components/ContentSidebar/ContentSidebar.js

```diff
@@ -226,10 +229,16 @@ class ContentSidebar extends PureComponent<Props, State> {
      * @param {Error} error - Error object
      * @return {void}
      */
-    errorCallback = (error: $AxiosXHR<any>): void => {
+    errorCallback = (error: Error): void => {
         /* eslint-disable no-console */
         console.error(error);
         /* eslint-enable no-console */
+
+        /* eslint-disable react/prop-types */
+        this.props.onError(error, ERROR_TYPE_CONTENT_SIDEBAR, ERROR_CODE_FETCH_FILE, {
+            [IS_ERROR_DISPLAYED]: true,
```

Comment:
It looks like this is already handled by `ErrorBoundary`?

Reply:
Can you explain further?

Reply:
Sure, sorry. It looks like ErrorBoundary already sets `IS_ERROR_DISPLAYED` to true for uncaught exceptions. Can ErrorBoundary also handle decorating the `onError` callback to set the value there as well? It looks like it's always set to true right now.